controllers/bulkApiMailer.js

```diff
@@ -0,0 +1,238 @@
+const MongoClient = require('mongodb').MongoClient;
+const chalk = require('chalk'); // To color message in console log
+
+const mailingManager = require('./mailing');
+const mailSend = require("../helpers/mailSend");
```

Comment:
This import does not exist currently; you should change it to leverage the export function you are doing further down.
```suggestion
const workerSendEmail = require("./workerSendEmail");
```